packages/core/src/modules/proofs/ProofsApi.ts

```diff
@@ -495,7 +495,7 @@ export class ProofsApi<PPs extends ProofProtocol[]> implements ProofsApi<PPs> {
           serviceParams: {
             service: recipientService.resolvedDidCommService,
             senderKey: ourService.resolvedDidCommService.recipientKeys[0],
-            returnRoute: true,
+            returnRoute: options.hasReturnRoute !== false, // defaults to true if missing
```

Comment:
I think this may be a bit easier to read. `??` will only take the second value if the first one is undefined (not if it's false like `||` does)

```suggestion
            returnRoute: options.hasReturnRoute ?? true
```